### PR TITLE
Shut down website

### PR DESCRIPTION
**Pull request by rmainwork**
This PR gets the repo ready to shut down the website (serf.io). However, I do have a few clarifying questions I'd like to call out.

1. There is one here that I'm a little unclear about and hasn't been updated - namely [`docs/internals/simulator.html.erb`](https://github.com/hashicorp/serf/blob/da2eb12071b319198e999aa2405a9bcf9ba4d1be/docs/internals/simulator.html.erb). 
   - Should this be converted to `.html.markdown` as well? It does look like an empty template that something is supposed to fill in, presumably with graphics of so
   - Is it considered "website code" (as-per 1e5befcb9db8bd1120a583bd2e4f5487d4a5ac86 and should be removed)
   - Should something else be done with it?
1. I also moved the `intro` directory under `/docs/intro` (updating paths as needed) since some of the documentation under `/docs` linked to it, and it seemed like it might contain important context for people digging in to the project for the first time. Is this okay, or should I remove it?

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207142155269478
  - https://app.asana.com/0/0/1207142155269473
  - https://app.asana.com/0/0/1207142155269482
  - https://app.asana.com/0/0/1207142155269475

### CODE REVIEW

File: demo/vagrant-cluster/README.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ At this point the two nodes are running and you can SSH in to play with them:
     ...
 
 To learn more about starting serf, joining nodes and interacting with the agent,
-checkout the [getting started guide](https://www.serf.io/intro/getting-started/install.html).
+checkout the [getting started guide](https://github.com/hashicorp/serf/docs/intro/getting-started/install.html).
```
[the remark this answers is not in the document]
Comment (rmainwork) [reply demo/vagrant-cluster/README.md:27]: Well spotted! Thanks! 🙌

**Comment by rmainwork** [comment]
@skpratt - if you wouldn't mind, I would appreciate a review. I think @byronwolfman's review was just to clean up the go linter errors for the most part. If you unassigned yourself because GH picked the wrong person, let me know and I can find someone else to take a look - cheers! 😄

**Comment by rboyer** [comment]
Will the parked domain have content specific permanent redirects left so permalinks go somewhere equivalent?

The consul and nomad websites deep links to the serf.io website:
```
consul $ git grep "serf.io"
content/docs/architecture/coordinates.mdx:All of this is provided through the use of the [Serf library](https://www.serf.io/).
content/docs/architecture/coordinates.mdx:[Serf's network coordinates here](https://www.serf.io/docs/internals/coordinates.html).
content/docs/architecture/gossip.mdx:through the [Serf library](https://www.serf.io/). The gossip protocol
content/docs/architecture/gossip.mdx:Refer to the [Serf documentation](https://www.serf.io/docs/internals/gossip.html) for additional information about the gossip protocol.
content/docs/architecture/gossip.mdx:are able to perform their functions by leveraging an embedded [Serf](https://www.serf.io/)
content/docs/architecture/gossip.mdx:[Serf gossip protocol guide](https://www.serf.io/docs/internals/gossip.html#lifeguard)
content/docs/install/glossary.mdx:Consul is built on top of [Serf](https://www.serf.io/) which provides a full
content/docs/security/encryption/index.mdx:The gossip protocol, as well as its membership management and message broadcasting features, use the [Serf library](https://www.serf.io/). 
content/docs/troubleshoot/faq.mdx:Consul uses the [Serf](https://www.serf.io) gossip protocol which relies on

nomad $ git grep "serf.io"
nomad/server.go:	// This value was tuned using https://www.serf.io/docs/internals/simulator.html to
website/content/docs/concepts/gossip.mdx:to manage membership. This is provided through the use of the [Serf library](https://www.serf.io/).
website/content/docs/concepts/gossip.mdx:with a few minor adaptations. There are more details about [Serf's protocol here](https://www.serf.io/docs/concepts/gossip.html).
website/content/docs/concepts/gossip.mdx:All of these features are provided by leveraging [Serf](https://www.serf.io/). It
website/content/docs/concepts/security.mdx:| **4648** / TCP + UDP | Servers | [gossip](/nomad/docs/concepts/gossip) protocol to manage server membership using [Serf](https://www.serf.io/).                                                                           |

```

Specifically:
- consul: https://www.serf.io/docs/internals/coordinates.html
- consul: https://www.serf.io/docs/internals/gossip.html
- nomad: https://www.serf.io/docs/internals/simulator.html
- nomad: https://www.serf.io/docs/concepts/gossip.html (lol dead already)

**Comment by rmainwork** [comment]
@rboyer I believe the parked domain will simply re-direct `serf.io` to the `/docs` directory in the GH repo. Any deep links would need to be updated in the relevant places(in this case, the consul and nomad docs)

**Comment by tkalus** [comment]
> @rboyer I believe the parked domain will simply re-direct `serf.io` to the `/docs` directory in the GH repo. I could be wrong but I believe any deep links would need to be updated in the relevant places(in this case, the consul and nomad docs)....

Confirmed that hc-centralized-dns parked domain capabilities are a simple "entire domain" HTTP 302 redirect; we don't have the capability to handle complex/per-page redirects with that mechanism.

**Comment by rboyer** [comment]
Are y'all going to take care of the followups in Consul and Nomad to avoid hitting the destination-removing 302 redirect for existing serf.io links?

**Comment by jmurret** [comment]
Here is a PR for the consul side: https://github.com/hashicorp/consul/pull/21797

**Comment by rmainwork** [comment]
@jmurret Just FYI - I took a quick look at that PR....I believe GH file links need to contain the `/blob` links directly to a file and `/tree` to link to a directory as well as the default branch name (i.e: `blob/main/docs/faq.md`). So, [github.com/hashicorp/consul/docs/faq.md](https://github.com/hashicorp/consul/docs/faq.md) goes to a 404, instead the link would be [github.com/hashicorp/consul/blob/main/docs/faq.md](https://github.com/hashicorp/consul/blob/main/docs/faq.md).

**Comment by jmurret** [comment]
thank you.  i was planning on test links once your PR was merged.  this helpful to know.

**Comment by tgross** [comment]
Nomad site PR here: https://github.com/hashicorp/nomad/pull/24114

**Pull request merged**